library_panel.go

```diff
@@ -101,22 +101,30 @@ func (c *Client) LibraryPanels() ([]LibraryPanel, error) {
 
 // LibraryPanelByUID gets a library panel by UID.
 func (c *Client) LibraryPanelByUID(uid string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/%s", uid))
+	resp := &LibraryPanelCreateResponse{}
+	path := fmt.Sprintf("/api/library-elements/%s", uid)
+
+	err := c.request("GET", path, nil, nil, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &resp.Result, err
 }
 
-// LibraryPanelByName gets a library panel by name.
+// LibraryPanelByName gets library panels by name.
 func (c *Client) LibraryPanelByName(name string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/name/%s", name))
-}
+	resp := &struct {
+		Result []LibraryPanel `json:"result"`
+	}{}
+	path := fmt.Sprintf("/api/library-elements/name/%s", name)
 
-func (c *Client) panel(path string) (*LibraryPanel, error) {
-	resp := &LibraryPanelCreateResponse{}
 	err := c.request("GET", path, nil, nil, &resp)
 	if err != nil {
 		return nil, err
 	}
 
-	return &resp.Result, err
+	return &(resp.Result[0]), err
```

Comment:
Before returning, could we error out if the len of the response isnt' 1

Reply:
Additionally I don't think the parenthesis are required here.

Reply:
does this look okay? 619c469caa59b9581727e4ebedc7849d67b6a46a